scripts/generate-artifacts.js

```diff
@@ -185,7 +185,7 @@ function main(appRootDir, outputPath) {
         console.log(
           '\n\n[Codegen] >>>>> Building react-native-codegen package',
         );
-        // [TODO(macOS GH#774) - use execFileSync to help keep shell commands clean
+        // [macOS - use execFileSync to help keep shell commands clean
```

Comment:
Remove the dash to stay consistent with the ones below?

Reply:
Good call! But also this file goes away in 0.71 I have learned :D I'll still remove the dash